api-client/api-client.gemspec

```diff
@@ -0,0 +1,39 @@
+require_relative 'lib/api/client/version'
+
+Gem::Specification.new do |spec|
+  spec.name          = "api-client"
```

Comment:
This should be named something which we can publish to rubygems in the future if we wish to (or even to a future private DfE gemserver)

```ruby
spec.name  = "get_into_teaching_api_client_faraday"
```

or

```ruby
spec.name  = "get_into_teaching_api_client_enhanced"
```

Reply:
Yeah good point, I'll get that updated 👍